src/main/java/com/microsoft/aad/msal4j/TokenCache.java

```diff
@@ -103,6 +110,25 @@ private static void mergeJsonObjects(JsonObject old, JsonObject update) {
         }
     }
 
+    private static void mergeRemovals(JsonObject old, JsonObject update, String parentKey) {
```

Comment:
I'm not seeing this being used? am I missing something?

Reply:
it is used in mergeJsonObjects function

Reply:
where?

Reply:
removed